src/plugins/interactive_setup/public/cluster_configuration_form.test.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { fireEvent, render, waitFor } from '@testing-library/react';
```

Comment:
__question:__ when I run this test I see "Warning: Failed prop type: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`." Is it something we should care about?

Reply:
Yeh, this is React trying to be helpful but not being helpful. The onChange handler is added to the form element but React doesn't realise this so shows this warning in dev.

Reply:
Ah, good to know, thanks!